Source/WebCore/page/RemoteFrame.cpp

```diff
@@ -51,7 +51,7 @@ Ref<RemoteFrame> RemoteFrame::createSubframeWithContentsInAnotherProcess(Page& p
 
 RemoteFrame::RemoteFrame(Page& page, UniqueRef<RemoteFrameClient>&& client, FrameIdentifier frameID, HTMLFrameOwnerElement* ownerElement, AbstractFrame* parent, Markable<LayerHostingContextIdentifier> layerHostingContextIdentifier)
     : AbstractFrame(page, frameID, ownerElement, parent)
-    , m_window(RemoteDOMWindow::create(*this, GlobalWindowIdentifier { Process::identifier(), WindowIdentifier::generate() }))
+    , m_window(RemoteDOMWindow::create(*this, GlobalWindowIdentifier { frameID.processIdentifier(), WindowIdentifier::generate() }))
```

Comment:
I'm not sure about this change.

Reply:
Updated based on the offline conversation